### PR TITLE
Dockerfile documentation

### LINKED ISSUE #34: Document the use of Dockerfile using Podman
Document the use of Dockerfile using Podman

### PR DESCRIPTION
**Pull request by khallnayak**
Created documentation for #33 against #34

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,4 +19,35 @@ A collection of self-contained and well-documented issues for newcomers to start
...
+
+### Docker
+
+#### Pre-requisites
+1. `docker` installed
+
+#### Steps
+1. Clone the repository by executing `git clone https://github.com/t0xic0der/fedora-easyfix.git`.
+2. Make the local repository your current working directory by executing `cd fedora-easyfix/`.
+3. Build the image `docker build -t fedora-easyfix:1.0 .`.
```
Comment (gridhead) [inline README.md]:
```suggestion
3. Build the image `docker build . -f Dockerfile -t fedora-easyfix:latest`.
```

File: README.md [hunk 2]
```diff
@@ -19,4 +19,35 @@ A collection of self-contained and well-documented issues for newcomers to start
...
+
+### Podman
+
+#### Pre-requisites
+1. `podman` installed
+
+#### Steps
+1. Clone the repository by executing `git clone https://github.com/t0xic0der/fedora-easyfix.git`.
+2. Make the local repository your current working directory by executing `cd fedora-easyfix/`.
+3. Build the image `podman build -t fedora-easyfix:1.0 -f Dockerfile`.
```
Comment (gridhead) [inline README.md]:
```suggestion
3. Build the image `podman build . -f Dockerfile -t fedora-easyfix:latest`.
```

File: README.md [hunk 3]
```diff
@@ -19,4 +19,35 @@ A collection of self-contained and well-documented issues for newcomers to start
...
+4. Run
+   - Run with default port options`podman run -it -d -p 9696:9696 fedora-easyfix:1.0`.
```
Comment (gridhead) [inline README.md]:
```suggestion
   - Run with default port options `podman run -it -p 9696:9696 fedora-easyfix:latest -4`.
```

File: README.md [hunk 4]
```diff
@@ -19,4 +19,35 @@ A collection of self-contained and well-documented issues for newcomers to start
...
+#### Pre-requisites
+1. `podman` installed
+
+#### Steps
+1. Clone the repository by executing `git clone https://github.com/t0xic0der/fedora-easyfix.git`.
+2. Make the local repository your current working directory by executing `cd fedora-easyfix/`.
+3. Build the image `podman build -t fedora-easyfix:1.0 -f Dockerfile`.
+4. Run
+   - Run with default port options`podman run -it -d -p 9696:9696 fedora-easyfix:1.0`.                                   
+   - Run with custom port options`podman run -it -d -p <Port number>:<Port number> fedora-easyfix:1.0 -4 -p <Port number>`.
```
Comment (gridhead) [inline README.md]:
```suggestion
   - Run with custom port options `podman run -it -p <port number>:<port number> fedora-easyfix:latest -4 -p <port number>`.
```

**Review by gridhead — CHANGES_REQUESTED** [review]
As the command-line flags are mostly the same for Podman and Docker, I would suggest you to have just one set of instructions which are applicable to both. Take for instance, explain the steps with Podman as a default and add a note that these would work just fine with Docker.

**Comment by khallnayak** [comment]
The changes have been made as per requested.

**Pull request closed without merging**